src/main/webapp/app/job/job-creation-form/job-creation-form.component.ts

```diff
@@ -494,8 +502,10 @@ export class JobCreationFormComponent {
 
       this.lastSavedData.set(currentData);
       this.savingState.set('SAVED');
-    } catch {
-      this.toastService.showErrorKey('toast.saveFailed');
+    } catch (err) {
+      this.savingState.set('FAILED');
+      const httpError = err as HttpErrorResponse;
+      this.toastService.showError({ summary: 'Error', detail: 'Failed to save job: ' + httpError.statusText });
```

Comment:
please do not change the toast error,
```suggestion
      this.toastService.showErrorKey('toast.saveFailed');
```
this was the most updated version

Reply:
done